data/Language/Pascal.yaml

```diff
@@ -72,8 +72,8 @@ keywords:
   - with
   - xor
 datatypes:
-  - string
-  - boolean
-  - integer
-  - real
-  - char
+  - boolean_pascal
+  - char_pascal
+  - integer_pascal
+  - real_pascal
+  - string_pascal   #TODO
```

Comment:
missing starting space in comment (comments)

*Origin: YAMLLintBear, Section: `yaml`.*

Reply:
ok